src/models/transfers.js

```diff
@@ -434,10 +447,7 @@ function * setTransfer (transfer, requestingUser) {
   }
 }
 
-function * getFulfillment (transferId, requestingUser) {
-  if (!requestingUser || !requestingUser.name) {
```

Comment:
So `getFulfillment` allows anyone now? When does a non-registered user need to get a fulfillment?